tests/Tests.elm

```diff
@@ -7,22 +7,38 @@ import Model
 import Update
 import Form exposing (Form)
 import Form.Error exposing (..)
+import Form.Field as Field
 
 
 all : Test
 all =
     describe "Initial example validation"
         [ test "has no output" <|
-            \_ -> equal (Form.getOutput validatedForm) Nothing
+            \_ -> equal Nothing (Form.getOutput validatedForm)
         , test "has errors on expected fields" <|
             \_ ->
                 equal
+                    (Form.getErrors validatedForm)
                     [ ( "email", InvalidString )
-                    , ( "profile.age", InvalidInt )
                     , ( "profile.role", InvalidString )
                     , ( "profile.superpower", InvalidString )
                     ]
-                    (Form.getErrors validatedForm)
+        , test "append, set then get field in list" <|
+            \_ ->
+                let
+                    ( name, value ) =
+                        ( "links.0.name", "Twitter" )
```

Comment:
when wrongly indexed, the input will be ignored. it was adding too much complexity to deal with holes in a list :)